Source/p_setup.c

```diff
@@ -1038,6 +1040,27 @@ static void P_CreateBlockMap(void)
 
 #endif // MBF_STRICT
 
+static void P_SetSkipBlockStart(void)
```

Comment:
Thank you! If now you could add a comment to the new code, I think we'll be done.